tensorboard/components/tf_ng_tensorboard/BUILD

```diff
@@ -45,13 +49,24 @@ tf_js_binary(
     ],
 )
 
-ng_module(
-    name = "ng_main",
+tf_ts_library(
+    name = "config",
```

Comment:
If we expect that config.dev.ts and config.prod.ts are only ever going to have the one config value that indicates if they're dev or prod, it seems simpler just to inline the setting of that value into main.ts and main.prod.ts.  Then we could just have a single file that is conditioned on dev-or-prod, rather than conditioning both config and main separately.

Reply:
Any thoughts on this?

Reply:
Generally, even if TypeScript does not complain about it, I want `import` statements to appear at the top of the module. This change would require setting constants or evaluating expressions before import statements.

(another embarrassing reason): empirically, I had hard time satisfying TSC when setting the constant in the main.ts. It complains about re-declaring the variable.